packages/react/src/components/Masthead/Masthead.js

```diff
@@ -97,46 +123,60 @@ const Masthead = ({ navigation, ...mastheadProps }) => {
   return (
     <HeaderContainer
       render={({ isSideNavExpanded, onClickSideNavExpand }) => (
-        <>
-          <Header aria-label="IBM" data-autoid="masthead">
-            <SkipToContent />
-            <HeaderMenuButton
-              aria-label="Open menu"
-              data-autoid="masthead__hamburger"
-              onClick={onClickSideNavExpand}
-              isActive={isSideNavExpanded}
-            />
-            <div className={className}>
-              <a href="https://ibm.com">
-                <IbmLogo data-autoid="masthead__logo" />
-              </a>
-            </div>
-            <div className={`${prefix}--header__search`}>
-              {navigation !== false ? (
-                <MastheadTopNav {...mastheadProps} navigation={mastheadData} />
-              ) : null}
-              <MastheadSearch />
-            </div>
+        <div
+          className={`${prefix}--masthead ${mastheadSticky}`}
+          ref={stickyRef}>
+          <div className={`${prefix}--masthead__l0`}>
+            <Header aria-label="IBM" data-autoid="masthead">
+              <SkipToContent />
+              <HeaderMenuButton
+                aria-label="Open menu"
+                data-autoid="masthead__hamburger"
+                onClick={onClickSideNavExpand}
+                isActive={isSideNavExpanded}
+              />
+              <div className={className}>
+                <a href="https://ibm.com">
+                  <IbmLogo data-autoid="masthead__logo" />
```

Comment:
I was actually debating where to put this since the logo is nothing but an svg and attaching an attribute to it for analytics purposes seems pointless to me. In this case, from an analytics perspective, we'd want to know if it was clicked. So maybe the proper place for it would be in the surrounding `<a />`?

cc: @jeffchew @annawen1 thoughts?

Reply:
I thought stable selectors were a little more than just analytics, but adding to the `<a />` tag sounds logical to me. Should the link be apart of that IBMLogo component?

Reply:
That's a good question. Would we ever have an instance where the IBM logo is used and it isn't linked? If not then yeah we should put the `<a />` there as well.

cc: @jeffchew @wonilsuhibm

Reply:
Confirmed with @wonilsuhibm that it should always be linked. Plus I see that the `FooterLogo` is done this way as well, so I'll just match it to be consistent.